app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/WorkoutRepository.java

```diff
@@ -0,0 +1,23 @@
+package edu.drexel.lapcounter.lapcounter.backend.Database;

```

Comment:
Since my code got pulled into master first, the Workouts Database code will conflict with mine as I had made some changes.  If you get yours to a state where the diff is just additions rather the conflict, it would make the merging much easier.